<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Agendamento</class>
 <widget class="QFrame" name="Form_Agendamento">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LC INFORMÁTICA | CADASTRO ATENDIMENTO</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../img/produtos-.png</normaloff>../../img/produtos-.png</iconset>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <widget class="QFrame" name="fr_FormProdutos">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>500</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777207</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFF;
border: none</string>
   </property>
   <widget class="QLabel" name="lb_FormProdutos">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>491</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: bold;

border-bottom: 2px solid #A2A2A2
}</string>
    </property>
    <property name="text">
     <string>CADASTRAR ATENDIMENTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FotoLogo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>301</width>
      <height>411</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Insira uma imagem</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #A2A2A2;
background: url('Images/icon/Photo.svg') center no-repeat 
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormNome">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>NOME</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Nome">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>85</y>
      <width>501</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>Nome  completo</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormCpf">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>215</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_Departamento">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
text-transform: uppercase
}
QComboBox:Focus {
border: 1px solid red;
}
 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 25px;
     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }
QComboBox::down-arrow {
     image: url(&quot;+self.resourcepath('Images/down.png')+&quot;);
 }
</string>
    </property>
    <item>
     <property name="text">
      <string>SELECIONE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GABINETE</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lb_FormTelefone">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>120</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>TELEFONE</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormEndereco">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>215</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>ENDEREÇO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Cpf">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>215</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; 
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="inputMask">
     <string>000.000.000-00</string>
    </property>
    <property name="text">
     <string>..-</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Telefone">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>150</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; 
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="inputMask">
     <string>(00) 00000-0000</string>
    </property>
    <property name="text">
     <string>() -</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Form_Atendimento">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: normal;

border-bottom: 2px solid #A2A2A2;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>ATENDIMENTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormDepartamento">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>DEPARTAMENTO</string>
    </property>
   </widget>
   <widget class="QFrame" name="fr_BotoesFormProdutos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>1000</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#E1DFE0;
border: none;</string>
    </property>
    <widget class="QPushButton" name="Bt_Can_Agendamento">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>0</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: #1E87F0;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
     </property>
     <property name="text">
      <string>CANCELAR</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/sair/delete_delete_exit_1577.png</normaloff>:/sair/delete_delete_exit_1577.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>75</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Bt_Salvar_agendamento">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>0</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: #7AB32E;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
     </property>
     <property name="text">
      <string>SALVAR</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/salvar/salvar.png</normaloff>:/salvar/salvar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_qtdeMin">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #7AB32E;
color: #FFF</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Endereco">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>501</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>Endereço ou localidade</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormAssunto">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>300</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>ASSUNTO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Assunto">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>330</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>Opcional</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Observacao">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>Opcional</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormObservacao">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>OBSERVAÇÃO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_qtdeMin_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #7AB32E;
color: #FFF</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #CFCFCF;
border: 1px solid #A2A2A2;
color: #000;
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Prefeitura municipal de santa terezinha</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>tx_Endereco</zorder>
   <zorder>lb_FormProdutos</zorder>
   <zorder>lb_FotoLogo</zorder>
   <zorder>lb_FormNome</zorder>
   <zorder>tx_Nome</zorder>
   <zorder>lb_FormCpf</zorder>
   <zorder>cb_Departamento</zorder>
   <zorder>lb_FormTelefone</zorder>
   <zorder>lb_FormEndereco</zorder>
   <zorder>tx_Cpf</zorder>
   <zorder>tx_Telefone</zorder>
   <zorder>lb_Form_Atendimento</zorder>
   <zorder>lb_FormDepartamento</zorder>
   <zorder>fr_BotoesFormProdutos</zorder>
   <zorder>lb_qtdeMin</zorder>
   <zorder>lb_FormAssunto</zorder>
   <zorder>tx_Assunto</zorder>
   <zorder>tx_Observacao</zorder>
   <zorder>lb_FormObservacao</zorder>
   <zorder>lb_qtdeMin_2</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tx_Nome</tabstop>
  <tabstop>tx_Cpf</tabstop>
  <tabstop>tx_Telefone</tabstop>
  <tabstop>tx_Endereco</tabstop>
  <tabstop>cb_Departamento</tabstop>
  <tabstop>tx_Assunto</tabstop>
  <tabstop>tx_Observacao</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
